git_hub_contributors/contributors.js

```diff
@@ -0,0 +1,61 @@
+const gitHubUrl = 'https://api.github.com/';
+const otherParams = {
+    headers: {
+        "content-type": "application/json; charset=UTF-8",
+        "Accept": "application/vnd.github.v3+json"
+    },
+    method: "GET"
+};
+
+function getContributors(repo) {
+    const repoUrl = gitHubUrl + 'orgs/' + repo + '/repos';
+    console.log('repoURL:', repoUrl);
+    const fetchedRepos = fetch(repoUrl, otherParams)
+        .then((response) => {
+            return response.json();
+        })
+        .then((result) => {
+            // console.log("first result:", result);
+            return result;
+        })
+        .then((repos) => {
+            for (let i = 0; i < 3; i++) {
+                // console.log(repos[i]);
+                const {contributors_url} = repos[i];
+                const {name} = repos[i];
+                // console.log(contributors_url);
+                contributor_info(contributors_url, name);
+            }
+        })
+        .catch((error) => {
+            console.log("BAD!", error);
+        });
+    // console.log("fetchedRepos:", fetchedRepos);
+}
+
+const repoContribMap = new Map();
```

Comment:
it is bad practice to have global variables that store some state.
what if the `getContributors` is invoked more than once?

Reply:
Should I create object inside fetch method and return it?